css/blocks/page.css

```
@@ -0,0 +1,56 @@
+.page {
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 1.5;
+  font-family: Inter, Avenir, Helvetica, Arial, sans-serif;
```

Зачем нужен этот список шрифтов?
```suggestion
  font-family: Inter, sans-serif;
```

У меня на Маке, например, если что встанет шрифт семейства SF, если будет сразу же семейство идти. То есть, либо брендовый шрифт, либо типичный маковский, привычный уже моему глазу.

А по этому списку видимо до Гельветики дойдет. Хотя, есть вероятность, что у меня например только Helvetica Neue или я вообще ее удалил (или если Винда), и тогда врубится Ариал, который в сравнении с системными шрифтами ужасен как визуально, так и по набору начертаний. За что же такое наказание людям в виде Ариала?

Там что у Винды довольно приятный Колибри (вроде, может сейчас уже нет), что на Маках SF или Гельветика. Да на Убунте стоит шрифт Убунту, который лучше Ариала =)
Так что в этих списках якобы похожих шрифтов толку почти никакого. А иногда от них только хуже